CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/home/HomeViewModel.kt

```diff
@@ -49,12 +49,14 @@ class HomeViewModel @Inject constructor(
 
     private fun fetchUsers() {
         showLoading()
-        viewModelScope.launch {
-            when (val result = getUsersUseCase.execute()) {
-                is UseCaseResult.Success -> _users.value = result.data
-                is UseCaseResult.Error -> _error.emit(result.exception.message.orEmpty())
+        execute(run {
+            {
```

Comment:
this seems redundant with the above `{` 🤔

Reply:
@sleepylee It's necessary for `run` block.
But I realize that we don't need to use it in this case. Just remove it.